src/index.js

```diff
@@ -0,0 +1,112 @@
+import './style.css';
+import change from './status';
+import {
+  addTask, editTask, updateTask, removeTask, clearCompleted,
+} from './crud';
+
+let taskList = [];
+const input = document.getElementById('input');
+const push = document.getElementById('push');
+
+function saveList(list) {
```

Comment:
Use arrow functions, they provide cleaner syntax and proper readable code.

Reply:
Thanks for the review. 👍🏼